src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java

```diff
@@ -88,45 +83,41 @@
 
     private static final int EXPECTED_VARIANTS = 300;
 
+    private static final String INPUT_VCF_ID = "1";
+
+    private static final String INPUT_STUDY_ID = "genotyped-job";
+
+    private static final String INPUT_FILE = "/small20.vcf.gz";
+
     @Rule
     public TemporaryMongoRule mongoRule = new TemporaryMongoRule();
 
-    @Autowired
-    private JobLauncherTestUtils jobLauncherTestUtils;
+    @Rule
+    public PipelineTemporaryFolderRule temporaryFolderRule = new PipelineTemporaryFolderRule();
 
     @Autowired
-    private JobOptions jobOptions;
-
-    private String input;
-    private String outputDir;
-    private String compressExtension;
-    private String dbName;
-    private String vepInput;
-    private String vepOutput;
+    private JobLauncherTestUtils jobLauncherTestUtils;
 
-    private static String opencgaHome = System.getenv("OPENCGA_HOME") != null ? System.getenv("OPENCGA_HOME") : "/opt/opencga";
+    private static String opencgaHome = System.getenv("OPENCGA_HOME") != null ? System
+            .getenv("OPENCGA_HOME") : "/opt/opencga";
 
     @Test
     public void fullGenotypedVcfJob() throws Exception {
-        jobOptions.getPipelineOptions().put(JobParametersNames.INPUT_VCF, getResource(input).getAbsolutePath());
-
         Config.setOpenCGAHome(opencgaHome);
-        mongoRule.getTemporaryDatabase(dbName);
-
-        //stats file init
-        VariantSource source = (VariantSource) jobOptions.getVariantOptions().get(VariantStorageManager.VARIANT_SOURCE);
-        File statsFile = new File(Paths.get(outputDir).resolve(VariantStorageManager.buildFilename(source))
-                + ".variants.stats.json.gz");
-        statsFile.delete();
-        assertFalse(statsFile.exists());  // ensure the stats file doesn't exist from previous executions
-
-        // annotation files init
-        File vepInputFile = new File(vepInput);
-        vepInputFile.delete();
-        assertFalse(vepInputFile.exists());
+        File inputFile = getResource(INPUT_FILE);
+        String dbName = mongoRule.getRandomTemporaryDatabaseName();
+
+        String outputDirStats = temporaryFolderRule.newFolder().getAbsolutePath();
+        String outputDirAnnotation = temporaryFolderRule.newFolder().getAbsolutePath();
+
+        File variantsStatsFile = new File(URLHelper.getVariantsStatsUri(outputDirStats, INPUT_STUDY_ID, INPUT_VCF_ID));
+        File sourceStatsFile = new File(URLHelper.getSourceStatsUri(outputDirStats, INPUT_STUDY_ID, INPUT_VCF_ID));
+        assertFalse(variantsStatsFile.exists());
+        assertFalse(sourceStatsFile.exists());
```

Comment:
These asserts and the ones below should not be necessary because we are using temporary folders.